js/researches/getSentenceBeginnings.js

```diff
@@ -45,11 +45,14 @@ module.exports = function( paper ) {
 	var sentences = getSentences( paper.getText() );
 	var sentenceBeginnings = sentences.map( function( sentence ) {
 		var words = getWords( stripSpaces( sentence ) );
-		var firstWord = removeNonWordCharacters( words[ 0 ] );
-		if ( firstWordExceptions.indexOf( firstWord ) > -1 ) {
-			firstWord += " " + removeNonWordCharacters( words[ 1 ] );
+		if( words.length > 0 ) {
```

Comment:
Consider changing this into an early return ( `words.length === 0` ). This will improve readability as we get rid of the extra nested if.